docs/concepts/policy/resource-quotas.md

```diff
@@ -46,9 +46,8 @@ Resource Quota support is enabled by default for many Kubernetes distributions.
 enabled when the apiserver `--admission-control=` flag has `ResourceQuota` as
 one of its arguments.
 
-Resource Quota is enforced in a particular namespace when there is a
-`ResourceQuota` object in that namespace.  There should be at most one
-`ResourceQuota` object in a namespace.
+Resource Quota is enforced in a particular namespace when there is at least one
+`ResourceQuota` object in that namespace.
```

Comment:
should we make more than one resourcequota in one namespaces, if there is conflict...? so I am not sure about it.

Reply:
I have tried it using v1.9 and it seems okay. I didn't spend time on reasoning the potential conflicts though.